### ISSUE TITLE
how many epochs to reach top performence on market1501?

### ISSUE THREAD

**Issue by bbbxixixixi** [layumi/Person_reID_baseline_pytorch#236]
I use this command to run train.py, other parameters remain unchanged
python train.py --gpu_ids 0 --name ft_ResNet50  --batchsize 32  --data_dir F:\data\market1501\pytorch --erasing_p 0.5 --PCB
after 20 epochs, the loss is still very high, from orignial 38 to 37-38, and precision is around 3% in training. Are they the right hyperparameters? like lr 0.05

**Comment by bbbxixixixi** [comment]
after 60 epoch, I got result Rank@1:0.290974 Rank@5:0.463777 Rank@10:0.544834 mAP:0.113355

**Comment by layumi** [comment]
Hi @bbbxixixixi 
It is weird. Could you check your Numpy Version first? 
Is the result correct on the baseline model without PCB?
